src/Tests/Toolchain/ToolchainProvider.test.ts

```diff
@@ -271,9 +278,10 @@ suite('Toolchain', function() {
       test('request setDefaultToolchain', function() {
         const provider = new ToolchainProvider();
         const bnodes = NodeBuilder.createBackendNodes();
-        assert.strictEqual(bnodes.length, 1);
-        assert.strictEqual(bnodes[0].label, backendName);
-        const tnodes = NodeBuilder.createToolchainNodes(bnodes[0]);
+        assert.strictEqual(bnodes.length, 2);
+        assert.strictEqual(bnodes[0].label, oneBackendName);
+        assert.strictEqual(bnodes[1].label, backendName);
+        const tnodes = NodeBuilder.createToolchainNodes(bnodes[1]);
```

Comment:
```suggestion
        // Ignore bnodes[0] because it is ONE Toolchain backend.
        const tnodes = NodeBuilder.createToolchainNodes(bnodes[1]);
```